docs/additional-package-support.md

```diff
@@ -28,10 +53,65 @@ pwa:
   # favicon_16_16: /favicon-16x16.png
   # mask_icon: /safari-pinned-tab.svg
 ```
+5. Create a `manifest.json` file in the `source/` directory.
+```json
+{
+    "name": "string", //Application full name
+    "short_name": "Junzhou", //Application short name
+    "theme_color": "#49b1f5", //Match the browser's address bar color
+    "background_color": "#49b1f5",//Background color when loading the app
+    "display": "standalone",//Preferred display mode Other options are: fullscreen, minimal-ui, browser
+    "scope": "/",
+    "start_url": "/",
+    "icons": [ //The array specifies the icons icon parameter, which is used to adapt to different devices (requires png, at least one icon of 192px*192px)
+        {
+          "src": "images/pwaicons/36.png", //The directory of the icon file needs to be created by itself in the source/ directory.
+          "sizes": "36x36",
+          "type": "image/png"
+        },
+        {
+            "src": "images/pwaicons/48.png",
+          "sizes": "48x48",
+          "type": "image/png"
+        },
+        {
+          "src": "images/pwaicons/72.png",
+          "sizes": "72x72",
+          "type": "image/png"
+        },
+        {
+          "src": "images/pwaicons/96.png",
+          "sizes": "96x96",
+          "type": "image/png"
+        },
+        {
+          "src": "images/pwaicons/144.png",
+          "sizes": "144x144",
+          "type": "image/png"
+        },
+        {
+          "src": "images/pwaicons/192.png",
+          "sizes": "192x192",
+          "type": "image/png"
+        },
+        {
+            "src": "images/pwaicons/512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+          }
+      ],
+      "splash_pages": null //Configure a custom launch animation.
+  }
+```
+
+You can also quickly create `manifest.json` via [Web App Manifest] (https://app-manifest.firebaseapp.com/). (Web App Manifest requires at least one 512*512 pixel icon)
+
+6. You can check if the PWA configuration is in effect and the configuration is correct through the `Chrome` plugin `Lighthouse`.
+    - Open the blog page
+    - Launch the `Lighthouse` plugin (the `Lighthouse` plugin requires at least one 512*512 pixel icon)
 
-You can create a `manifest.json` fastly by visiting [Web App Manifest](https://app-manifest.firebaseapp.com/)
+For more on PWA (Progressive Enhanced Web Applications), see [Google Tools for Web Developers] (https://developers.google.com/web/tools/lighthouse/audits/address-bar)
```

Comment:
![image](https://user-images.githubusercontent.com/12621342/52514739-616bbd00-2c4f-11e9-8688-c5bc40f9a274.png)

There is an extra space here

Reply:
ok.

Reply:
Still an extra space here?

Reply:
Chinese version is OK while English version is error

Reply:
> Chinese version is OK while English version is error

Sorry, I executed git add without saving changes in vscode. I just found this stupid error and wondered why the change didn't take effect.

Reply:
@Molunerfinn Now I fixed these errors, please check